www-client/firefox/firefox-48.0.ebuild

```diff
@@ -54,11 +54,12 @@ SRC_URI="${SRC_URI}
 ASM_DEPEND=">=dev-lang/yasm-1.1"
 
 RDEPEND="
-	>=dev-libs/nss-3.23
+	>=dev-libs/nss-3.24
 	>=dev-libs/nspr-4.12
 	selinux? ( sec-policy/selinux-mozilla )"
 
 DEPEND="${RDEPEND}
+	system-sqlite? ( >=dev-db/sqlite-3.12.2 )
```

Comment:
Please try to keep things sorted. I think the arch-related flags were intentionally listed last but other flags should be sorted lexically, i.e. system-sqlite after pgo.